docs/list.html

```diff
@@ -0,0 +1,189 @@
+          </li>
+          <li>
+            <a href="/docs/navigation.html" class="navlink">Navigation</a>
+          </li>
+          <li>
+            <a href="/docs/input.html" class="navlink">Input</a>
+          </li>
+          <li>
+            <a href="/docs/display.html" class="navlink">Display</a>
+          </li>
+          <li>
+            <a href="/docs/ratings.html" class="navlink">Ratings</a>
+          </li>
+          <li>
+            <a href="/docs/list.html" class="navlink">List</a>
+          </li>
+        </nav>
+      </aside>
+      <section id="list" class="mb-2 p-1 flex-1">
+        <header class="border-bottom">
+          <h1>List</h1>
+          <p class="py-1">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          </p>
+        </header>
+
+        <article>
+          <h5 class="my-2">Stacked List</h5>
+          <div class="section-border">
+            <ul class="text-center mb-2">
+              <li class="bg-success p-2 text-12 my-1">
+                Success! Profile updated successfully
+              </li>
+              <li class="bg-danger p-2 text-12 my-1">
+                Oops! Couldn't update Profile
+              </li>
+              <li class="bg-warning p-2 text-12 my-1">
+                Are you sure want to update your account?
+              </li>
+            </ul>
+            <code class="iframe-container">
+              <iframe
+                width="500px"
+                src="https://carbon.now.sh/embed?bg=rgba%28250%2C250%2C250%2C1%29&t=one-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=0px&dsblur=100px&wc=true&wa=true&pv=71px&ph=98px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cul%2520class%253D%2522text-center%2522%253E%250A%2520%2520%253Cli%2520class%253D%2522bg-primary%2520p-1%2522%253EList%2520Item%25201%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522bg-secondary%2520p-1%2522%253EList%2520Item%25203%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522bg-success%2520p-1%2522%253EList%2520Item%25203%253C%252Fli%253E%250A%253C%252Ful%253E"
+                sandbox="allow-scripts allow-same-origin"
+                class="h-28 w-full border-none d-block"
+              >
+              </iframe>
+            </code>
+          </div>
+        </article>
+
+        <article>
+          <h5 class="my-2">Simple List</h5>
+          <div class="section-border">
+            <div class="my-2">
+              <ul class="d-flex flex-col items-center">
+                <li class="d-flex items-center">
+                  <span class="fas fa-circle d-block"></span>
+                  <p class="text-12 ml-1">Item 1</p>
+                </li>
+                <li class="d-flex items-center">
+                  <span class="fas fa-circle d-block"></span>
+                  <p class="text-12 ml-1">Item 2</p>
+                </li>
+                <li class="d-flex items-center">
```

Comment:
Instead of repeating same property you can directly add that in **li** in your list.css or you can make separate class and use it

Reply:
Thanks for the suggestion. But this atomic classes are the intentional so that they can be used in combination with other classes.